media/transcoder/create_job_with_periodic_images_spritesheet.go

```diff
@@ -0,0 +1,117 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package transcoder
+
+// [START transcoder_create_job_with_periodic_images_spritesheet]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"github.com/golang/protobuf/ptypes/duration"
+
+	transcoder "cloud.google.com/go/video/transcoder/apiv1beta1"
+	transcoderpb "google.golang.org/genproto/googleapis/cloud/video/transcoder/v1beta1"
+)
+
+// createJobWithPeriodicImagesSpritesheet creates a job from an ad-hoc configuration and generates two spritesheets from the input
+// video. Each spritesheet contains images that are captured periodically based on a user-defined time interval.
```

Comment:
Might want to wrap these comments so they're a bit more narrow and embed more nicely in docs.

I just added this style guide entry: https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#line-length

Reply:
Done